enterprise/security/src/main/java/org/neo4j/server/security/enterprise/configuration/SecuritySettings.java

```diff
@@ -234,7 +254,10 @@
             ( logs ) -> new File( logs, "security.log" ),
             PATH );
 
-    @Description( "Set to log successful authentication events." )
+    @Description( "Set to log successful authentication events to the security log." +
+                  "If this is set to `false` only failed authentication events will be logged, which " +
+                  "could be useful if you find that the successful events spam the logs too much, " +
+                  "and you do not require full auditing capability")
```

Comment:
@jjaderberg Judging by the looks of the `@Description`s in `GraphDatabaseSettings`, that is _not_ current styling. It seems these are optimised for code readability, and to fit on 120 chars.